CommonKit/Sources/CommonKit/Services/KeychainStore.swift

```diff
@@ -14,7 +14,18 @@ public final class KeychainStore: SecuredStore {
     // MARK: - Properties
     private static let keychain = Keychain(service: "\(AdamantSecret.appIdentifierPrefix).im.adamant.messenger")
     
-    public init() {}
+    private let secureStorage: SecureStorageProtocol = AdamantSecureStorage()
```

Comment:
Let's use DI. In the extension we can initialize `AdamantSecureStorage` on initializing `KeychainStore`